### ISSUE TITLE
Error: No default engine was specified and no extension was provided node.js

### ISSUE THREAD

**Issue by carlosandresSanchez** [Smile-SA/node-pushserver#48]
Help! Why does this happen when I call subscribe?

**Comment by marob** [comment]
What installation method have you used?
I suspect the "npm install" yo have (partially) failed. Can you install again?

**Comment by carlosandresSanchez** [comment]
Well, I reinstall with npm and nothing. It's strange. I had used both methods. 

I tried uinstall with npm uninstall and later install only with git and occurs the same.

**Comment by carlosandresSanchez** [comment]
I get this log.


npm WARN deprecated lodash@1.3.1: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
|
kerberos@0.0.3 install /usr/local/lib/node_modules/node-pushserver/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos
 (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/kerberos/lib/kerberos.o

 bson@0.2.2 install /usr/local/lib/node_modules/node-pushserver/node_modules/mongoose/node_modules/mongodb/node_modules/bson
 (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bson/ext/bson.o
/usr/local/bin/pushserver -> /usr/local/lib/node_modules/node-pushserver/bin/pushserver.js
node-pushserver@0.5.4 /usr/local/lib/node_modules/node-pushserver
├── commander@2.7.1 (graceful-readlink@1.0.1)
├── lodash@1.3.1
├── express@3.2.6 (methods@0.0.1, fresh@0.1.0, range-parser@0.0.4, cookie-signature@1.0.1, buffer-crc32@0.2.1, cookie@0.1.0, commander@0.6.1, mkdirp@0.3.4, send@0.1.0, debug@2.2.0, connect@2.7.11)
├── apn@1.7.5 (q@1.4.1, debug@2.2.0, node-forge@0.6.38)
├── node-gcm@0.13.1 (debug@0.8.1, lodash@3.10.1, request@2.67.0)
└── mongoose@3.6.20 (regexp-clone@0.0.1, muri@0.3.1, sliced@0.0.5, hooks@0.2.1, mpath@0.1.1, mpromise@0.2.1, ms@0.1.0, mongodb@1.3.19)*

**Comment by marob** [comment]
Do you still have installation problem?

**Comment by CoreSoft2** [comment]
I see same issue. any resolution?

**Comment by carlosandresSanchez** [comment]
Yes, i do. I still have the problem. I don't understand what is the problem.

**Comment by CoreSoft2** [comment]
OK, the problem is not with push server but the way error handing is implemented (rather not implemented), if the data is not in a correct format or database connection is down, the express tries to render a error page. that is what you see. There is no view in the push server.

**Comment by carlosandresSanchez** [comment]
Ok, I undertand. One more question. Is it needed to configurate the APN certificates to subscribe a device ? I was trying to make the subscription with invented values, Perhaps that is the problem.
